disruptor.py

```diff
@@ -134,9 +136,11 @@ async def start(self):
         self.manual_user_ratelimits = defaultdict(lambda: ManualRateLimit(
             rate=float(self.config["user_ratelimit.rate"]),
             per=float(self.config["user_ratelimit.per"])))
+        self.disrupt_probability = float(self.config["disrupt_probability"])
```

Comment:
I think this would make more sense as an input to the `MonologueInfo` constructor.